### PR TITLE
Additional test coverage

### PR DESCRIPTION
**Pull request by bamandel**

### CODE REVIEW

File: src/main/java/com/blackducksoftware/integration/hub/alert/audit/repository/AuditNotificationRepositoryWrapper.java [hunk 1]
```diff
@@ -43,10 +43,6 @@ public AuditNotificationRepositoryWrapper(final AuditNotificationRepository repo
         return decryptSensitiveData(getRepository().findByAuditEntryId(auditEntryId));
     }
 
-    public List<AuditNotificationRelation> findByNotificationId(final Long notificationId) {
```
Comment (gkillough) [inline src/main/java/com/blackducksoftware/integration/hub/alert/audit/repository/AuditNotificationRepositoryWrapper.java]: Unused?

File: src/test/java/com/blackducksoftware/integration/hub/alert/AlertConstantsTest.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import java.lang.reflect.InvocationTargetException;
+
+import org.junit.Test;
+
+import javassist.Modifier;
+
+public class AlertConstantsTest {
+
+    @Test
+    public void testInstantiationException() throws NoSuchMethodException, SecurityException {
```
Comment (gkillough) [inline src/test/java/com/blackducksoftware/integration/hub/alert/AlertConstantsTest.java]: Why are we testing reflective constructors? Is this one of those weird edge cases where the class has no defined methods, but still claims line coverage is incomplete?

File: src/test/java/com/blackducksoftware/integration/hub/alert/audit/repository/AuditEntryRepositoryWrapperTest.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        outputLogger = new OutputLogger();
+    }
+
+    @After
+    public void cleanup() throws IOException {
+        outputLogger.cleanup();
+    }
+
+    @Test
+    public void testFindFirstByCommonConfigIdOrderByTimeLastSentDesc() throws IOException {
```
Comment (gkillough) [inline src/test/java/com/blackducksoftware/integration/hub/alert/audit/repository/AuditEntryRepositoryWrapperTest.java]: I think we want to test custom queries with a live database.

File: src/test/java/com/blackducksoftware/integration/hub/alert/channel/email/template/FreemarkerTargetTest.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Test;
+
+public class FreemarkerTargetTest {
+
+    @Test
+    public void testTreemarker() {
```
Comment (gkillough) [inline src/test/java/com/blackducksoftware/integration/hub/alert/channel/email/template/FreemarkerTargetTest.java]: Typo?

File: src/test/java/com/blackducksoftware/integration/hub/alert/audit/repository/AuditEntryRepositoryWrapperTest.java (hunk 3 above)
Comment (gkillough) [reply src/test/java/com/blackducksoftware/integration/hub/alert/audit/repository/AuditEntryRepositoryWrapperTest.java]: If we already do this in the repository test, feel free to ignore. This seems fine for the wrapper test.

File: src/test/java/com/blackducksoftware/integration/hub/alert/AlertConstantsTest.java (hunk 2 above)
Comment (psantos1113) [reply src/test/java/com/blackducksoftware/integration/hub/alert/AlertConstantsTest.java]: This seems odd because we hide the default constructor in AlertConstants because it should only contain static fields.  We throw the exception as a precaution.  That's what he's trying to test here.

File: src/test/java/com/blackducksoftware/integration/hub/alert/audit/repository/AuditEntryRepositoryWrapperTest.java (hunk 3 above)
Comment (bamandel) [reply src/test/java/com/blackducksoftware/integration/hub/alert/audit/repository/AuditEntryRepositoryWrapperTest.java]: These tests are for testing exceptions specifically, so a DB connection isn't required. I can try to find the accompanied IT if it exists and add it there, but right now this runs as a unit test as opposed to an IT
